dags/prio/dataproc.py

```diff
@@ -0,0 +1,78 @@
+from airflow import DAG
+from airflow.contrib.hooks.gcp_api_base_hook import GoogleCloudBaseHook
+from airflow.contrib.operators.dataproc_operator import (
+    DataprocClusterCreateOperator,
+    DataprocClusterDeleteOperator,
+    DataProcPySparkOperator,
+)
+
+
+def spark_subdag(
```

Comment:
I'd rather follow up with a separate PR instead of expanding the scope of this one. I don't think the dataproc runner was available when I started this PR.